keyboards/ploopyco/trackball_thumb/trackball_thumb.h

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2021 Colin Lam (Ploopy Corporation)
+ * Copyright 2020 Christopher Courtney, aka Drashna Jael're  (@drashna) <drashna@live.com>
+ * Copyright 2019 Sunjun Kim
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+#include "analog.h"
+#include "opt_encoder.h"
+#if defined(KEYBOARD_ploopyco_trackball_thumb_rev1_001)
+#    include "rev1_001.h"
+#endif
+
+// Sensor defs
+#define OPT_ENC1 F4
+#define OPT_ENC2 F0
+#define OPT_ENC1_MUX 4
+#define OPT_ENC2_MUX 0
+
+bool encoder_update_kb(uint8_t index, bool clockwise);
+bool encoder_update_user(uint8_t index, bool clockwise);
+
+#define LAYOUT(BFT, BF, BL, BM, BR, BB) \
+    { {BFT, BF, BL, BM, BR, BB}, }
```

Comment:
Would it be possible to change the layout to

```
    BFT, BL, BM, BR, BB, 
    BF
```

I think this represents the physical layout of the device better. While messing with layers I've found it really hard to wrap my head around the layout as it is currently.

Reply:
Definitely can do!